examples/config.yml

```diff
@@ -81,6 +81,9 @@ backplane:
 worker:
   port: 8981
   publicName: "localhost:8981"
+  grpcMetrics:
```

Comment:
I think we should pull this config up to the top level so that it can control both server and worker together instead of duplicating it.

Reply:
Not everyone wants to instrument both: I am planning on not instrumenting workers w/histograms, but may change my mind, so I think we should allow knobs for server and worker to be separate. An example is that someone may want to use metrics for SLA enforcement only, and they wont want to take a performance hit for adding metrics to workers.

Reply:
You'll still be able to configure them separately by providing a different config file to server/worker. The example file has both configurations but only for simplicity.